doc-content/shared-kie-docs/src/main/asciidoc/Install/testing-clustering-environment-proc.adoc

```diff
@@ -0,0 +1,122 @@
+[id='testing-clustering-environment-proc_{context}']
+= Testing that your infrastructure meets the minimum hardware and performance requirements for a viable high availability (HA) on-premises environment.
```

Comment:
Need to trim the title imo, plus a stray period in the title. How about just "Testing your high availability (HA) on-premise infrastructure". Based on the first sentence of the module, I think that title is sufficient for what's going on and shortens it a lot.

Reply:
Agreed and done.